"not all trait items implemented" rust error when running yarn run build

Howdy,

I cloned the repo, ran `yarn install` and then `yarn run build` and received the following rust error:

```
error[E0046]: not all trait items implemented, missing: `change`
  --> src/app.rs:46:1
   |
46 | impl Component for App {
   | ^^^^^^^^^^^^^^^^^^^^^^ missing `change` in implementation
   |
   = help: implement the missing item: `fn change(&mut self, _: <Self as yew::html::Component>::Properties) -> bool { todo!() }`
```

Implementing the missing method fixes the issue and the crate then compiles and `yarn run start:dev` starts successfully.

PR incoming.

--- a/src/app.rs
+++ b/src/app.rs
@@ -47,6 +47,10 @@ impl Component for App {
     type Message = Msg;
     type Properties = ();
 
+    fn change(&mut self, _: <Self as yew::html::Component>::Properties) -> bool {
+        todo!()
+    }
+
     fn create(_: Self::Properties, link: ComponentLink<Self>) -> Self {
         let storage = StorageService::new(Area::Local).unwrap();
         let entries = {

--- a/src/app.rs
+++ b/src/app.rs
@@ -47,10 +47,6 @@ impl Component for App {
     type Message = Msg;
     type Properties = ();
 
-    fn change(&mut self, _: <Self as yew::html::Component>::Properties) -> bool {
-        todo!()
-    }
-
     fn create(_: Self::Properties, link: ComponentLink<Self>) -> Self {
         let storage = StorageService::new(Area::Local).unwrap();
         let entries = {
@@ -67,6 +63,10 @@ impl Component for App {
             edit_value: "".into(),
         };
         App { link, storage, state }
+    }
+
+    fn change(&mut self, _props: Self::Properties) -> ShouldRender {
+        false
     }
 
     fn update(&mut self, msg: Self::Message) -> ShouldRender {

